config/go-apiary.yaml

```diff
@@ -0,0 +1,5 @@
+libraries:
```

Comment:
LOL.  About 10 seconds ago, we moved node-apiary.yaml out of config and into .kokoro-autosynth.  Could you do the same with this file?

Reply:
Done. Thanks, this is a better/less confusing location.